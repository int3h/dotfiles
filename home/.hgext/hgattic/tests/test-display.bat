@echo off

rem setup hgattic in a temp test directory
setlocal
cd ..
cd > tmpfile
set /p atticdir= < tmpfile
erase tmpfile
cd tests
mkdir test
cd test
call hg init
cd .hg
echo [extensions] >tmp
echo hgattic=%atticdir%\attic.py >>tmp
move tmp hgrc
cd ..

rem replace here with test content
echo a > a.txt
call hg addrem
call hg shelve -q a
echo b > b.txt
call hg addrem
call hg shelve -q b
echo ### hg attic
call hg attic
rem end test content

rem cleanup
cd ..
rmdir /S /Q test
endlocal
