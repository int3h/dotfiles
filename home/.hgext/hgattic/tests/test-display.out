 adding a.txt
adding b.txt
### hg attic
   a
 C b

